tests/demo_data/generic_labels.ris

```diff
@@ -22,6 +22,7 @@ SN  - 1932-6208
 M1  - 1008150341
 L2  - http://example.com
 UR  - http://example_url.com
+N1  - ASReview_relevant
 LI  - 1
```

Comment:
Remove this one?

Reply:
I reverted the initial commit to adjust the file: https://github.com/asreview/asreview/pull/603/commits/d026094cb1b7e3a4070b1354c4744166260638c3